modules/conflict_resolver/jsx/fix_conflict_form.js

```diff
@@ -0,0 +1,85 @@
+import {Component} from 'react';
+import PropTypes from 'prop-types';
+
+class FixConflictForm extends Component {
+  constructor(props) {
+    super(props);
+
+    this.fix = this.fix.bind(this);
+  }
+
+  fix(e) {
```

Comment:
Hey @xlecours fix(e){...} is missing the function comment.

Reply:
@maltheism Should be addressed